apps/complaints/views/confirm.html

```diff
@@ -0,0 +1,8 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{#tableSections}}
+      {{> partials-summary-table}}
+    {{/tableSections}}
+    {{#input-submit}}confirm{{/input-submit}}
+  {{/page-content}}
+{{/partials-page}}
```

Comment:
Is this file included in hof-template-partials? Is so could we use that? No worries if not, will make a note to add it